plugin/db/mysql/mysql.go

```diff
@@ -50,7 +55,7 @@ type Driver struct {
 	connectionCtx db.ConnectionContext
 	dbType        db.Type
 
-	db *sql.DB
+	DB *sql.DB
```

Comment:
We have GetDbConnection() to get this db.

Reply:
The implementation of `GetDbConnection()` in Postgres and SQLite has side effects.
IMO we should add a `SwitchDB()` to change the connected database for the internal `*sql.DB`, and leave `GetDbConnection()` stateless.
WDYT?

Reply:
We can add a TODO for MySQL. GetDbConnection() means to do a switchDB() + getDBConnection().

Reply:
I prefer `GetDbConnection()` just returns the `*sql.DB` and does not change it, and `SwitchDB()` changes the `*sql.DB`. It will make the API semantic clear.
IMO we should prevent unexpected side effects.

Reply:
If GetDbConnection works for now, let's use it. It's better not to expose DB internal object here.

Yeah, maybe GetDbConnection() should create and return a new db connection object and return it without mutating existing db in the driver.